### DISCUSSION TITLE
Errors in GND calculation

### DISCUSSION THREAD

**Discussion by BettyWalzier** [mtex-toolbox/mtex#681] [Ask Anything] [answered]
Hi all,

Today I use the code for GND density calculation on the MTEX website for copper: https://mtex-toolbox.github.io/GND.html.
When it runs to the line of 
**% consider only the Fe(alpha) phase
ebsd = ebsd('indexed').gridify;**
It gives an error as follows:
![image](https://user-images.githubusercontent.com/76571114/103769426-85470180-5024-11eb-8b06-a352e02b7f5d.png)
Is there anyone who knows how to solve this problem? I run the code for Fe with FCC structure, and it runs perfectly before.

Thanks in advance,
Best regards,
Betty

**Comment by ralfHielscher** [comment, marked as answer]
Hi Betty,

most likely you are not using the current MTEX version 5.5.

Ralf.

**Reply by BettyWalzier** [reply]
Hi Ralf,

Thanks for your reply. 
I will try to use MTEX version 5.5. Then I will give you the result.

Best regards,
Betty

**Reply by BettyWalzier** [reply]
Hi Ralf,

Update for the problem. It has been solved using the version 5.5. Thanks a lot!

Best regards,
Betty
